lookout/style/format/feature_extractor.py

```diff
@@ -337,6 +335,24 @@ def select_features(self, X: numpy.ndarray, y: numpy.ndarray) -> Tuple[numpy.nda
         self._compute_feature_info()
         return X, self.selected_features
 
+    def get_composite_class_representations(self) -> List[str]:
```

Comment:
if we have it in FE it can probably be a property

Reply:
It does some non-trivial computations inside so it is scary; it would also remove the harmony with the second "composite" method I moved.